src/topology/bornology/hom.lean

```diff
@@ -0,0 +1,119 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import topology.bornology.basic
+
+/-!
+# Locally bounded maps
+
+This file defines locally bounded maps between bornologies.
+
+We use the `fun_like` design, so each type of morphisms has a companion typeclass which is meant to
+be satisfied by itself and all stricter types.
+
+## Types of morphisms
+
+* `locally_bounded_map`: Locally bounded maps. Maps which preserve boundedness.
+
+## Typeclasses
+
+* `locally_bounded_map_class`
+-/
+
+open bornology filter function
+
+variables {F α β γ δ : Type*}
+
+/-- The type of bounded maps from `α` to `β`, the maps which send a bounded set to a bounded set. -/
+structure locally_bounded_map (α β : Type*) [bornology α] [bornology β] :=
```

Comment:
```suggestion
structure locally_bounded_map (α β) [bornology α] [bornology β] :=
```
I am pretty sure you don't need the `Type*` there.

Reply:
It is good practice to have it, if only to demonstrate to the reader that it's not `Sort*`